Content.Shared/Item/ItemComponent.cs

```diff
@@ -66,6 +65,38 @@ public VisualsChangedEvent(NetEntity item, string containerId)
     }
 }
 
+/// <summary>
+/// Abstracted sizes for items.
+/// Used to determine what can fit into inventories.
+/// </summary>
+public enum ItemSize
+{
+    /// <summary>
+    /// Items that can be held completely in one's hand.
+    /// </summary>
+    Tiny = 1,
+
+    /// <summary>
+    /// Items that can fit inside of a standard pocket.
+    /// </summary>
+    Small = 2,
+
+    /// <summary>
+    /// Items that can fit inside of a standard bag.
+    /// </summary>
+    Normal = 3,
+
+    /// <summary>
+    /// Items that are too large to fit inside of standard bags, but can worn in exterior slots or placed in custom containers.
+    /// </summary>
+    Bulky = 4,
+
+    /// <summary>
+    /// Items that are too large to place inside of any kind of container.
+    /// </summary>
+    Large = 5
```

Comment:
IMO it's not clear that "Large" is larger than "Bulky", I would've assumed the opposite. Maybe we should use Large & Huge instead?

Reply:
I think dnd uses Large -> Huge instead so might make more sense.

Also wouldn't it make more sense for the enum values to be the weight? Or alternatively use prototypes.